boards/arm/stm32h7b3i_dk/stm32h7b3i_dk.dts

```diff
@@ -49,6 +50,13 @@
 		zephyr,memory-region = "SDRAM2";
 	};
 
+	transceiver0: can-phy0 {
+		compatible = "microchip,mcp2562fd", "can-transceiver-gpio";
+		standby-gpios = <&gpioh 8 GPIO_ACTIVE_HIGH>;
+		max-bitrate = <5000000>;
+		#phy-cells = <0>;
+	};
```

Comment:
I'm not finding this as part of BSP. Shouldn't it be moved to a sample overlay ?

Reply:
ok, found as "MC**D**2562FD" in the user manual.
Well, as long as it is clearly documented, which is the case, I'm fine.